src/components/ApplicationDetail/ApplicationQnAItem/ApplicationQnAItem.component.tsx

```diff
@@ -17,7 +18,18 @@ const ApplicationQnAItem = ({
     <Styled.ApplicationQnAItemContainer>
       <Styled.Title>{unescape(content)}</Styled.Title>
       {description && <Styled.Description>{unescape(description)}</Styled.Description>}
-      {answer.content && <Styled.Answer>{unescape(answer.content)}</Styled.Answer>}
+      {answer.content && (
+        <Styled.Answer>
+          <Linkify
+            // eslint-disable-next-line react/no-unstable-nested-components
+            componentDecorator={(decoratedHref: string, decoratedText: string, key: number) => (
+              <a href={decoratedHref} key={key} target="_blank" rel="noreferrer">
+                {decoratedText}
+              </a>
+            )}
+          >{`${unescape(answer.content)} http://dididy.io`}</Linkify>
```

Comment:
```suggestion
          >{unescape(answer.content)}</Linkify>
```

http://dididy.io 도 필요한 부분일까요?

Reply:
@minsour 테스트용 텍스트인데 제거했습니다!